exercise9_beyond1.py

```diff
@@ -0,0 +1,8 @@
+user_input = float(input())
+
+def square(number):
```

Comment:
It is completely different from Beyond Excercise #9 task 1 requirements. Also, I can see the function takes `number` but never uses it.